redbot/core/commands/commands.py

```diff
@@ -66,6 +70,10 @@
 DisablerDictType = MutableMapping[discord.Guild, Callable[["Context"], Awaitable[bool]]]
 
 
+class RedUnhandledAPI(Exception):
```

Comment:
I have mixed feelings on this one, and will take the time to write it out in more detail later.

Reply:
Lack of being in `__all__` already prevents the name from being exported. I don't want to document this in details yet, but I do intend to make this public when implementing the data *fetching* API later on in case a cog wants to send this in particular cases it doesn't handle yet on that, when it does handle other cases (since fetching and making a display has more to it than just deleting what's appropriate to delete)

Reply:
It's in all and sparsely documented now